salt/journal-cms/init.sls

```diff
@@ -365,22 +365,6 @@ journal-cms-defaults-users-{{ username }}:
             - migrate-content
 {% endfor %}
 
-{% for process in ['article-import', 'send-notifications'] %}
```

Comment:
moved from init.sls to processes.sls purely to couple the service definition with the service itself